query02.sql

```diff
@@ -7,9 +7,10 @@
 */
 
 -- Enter your SQL query here
-select ...
-
+select (((206354 - count(*)) * 100) / 206453) :: text || '%' AS percent_difference
```

Comment:
Careful with numeric division. While this will give a nice round answer, you do lose some precision in keeping everything as integers.